paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -853,6 +853,9 @@ class PoolingMKLDNNHandler : public MKLDNNHandlerT<T, mkldnn::pooling_forward,
         CorrectOutputSize(src_tz, dst_tz, ksize, paddings, strides,
                           mkldnn_paddings[1]);
       }
+
```

Comment:
1. why `ctx.Attr<bool>("adaptive")` is not here but inside the function `ComputeAdaptivePoolParameters`?

2. In the function `ComputeAdaptivePoolParameters` declaration, it is not clear what will be changed, what will not. Maybe we could refer to tester_helper.cc
```
void TestOneThreadPrediction(const PaddlePredictor::Config *config, const std::vector<std::vector<PaddleTensor>> &inputs, 
 std::vector<std::vector<PaddleTensor>> *outputs, int num_threads, bool use_analysis = FLAGS_use_analysis)
```
in this way from the `ComputeAdaptivePoolParameters(ctx, src_tz, &ksize, &strides);`, we know what will be changed what will not.
But maybe point 2 is not important

Reply:
ComputeAdaptivePoolParameters is used in two places e.g. once inside PoolMKLDNNHandler and once directly in PoolGrad. So it was easiest for me to just read it inside rather than read it in two places and then pass as argument which would introduced tiny overhead as more data is passed to call.

Reply:
Ok, it is clear. Thanks